go/ct/gen/code.go

```diff
@@ -114,6 +114,12 @@ func (g *CodeGenerator) Generate(assignment Assignment, rnd *rand.Rand) (*st.Cod
 		minSize += size
 	}
 
+	// If there are any variables that need to be bound to code, there must be at
+	// least on instruction in the resulting code.
+	if minSize == 0 && len(g.varIsCodeConstraints) > 0 {
```

Comment:
We will merge this in my pr soon, where I have the test for it. 
if you get the chance, remove it. otherwise we will rebase it.